hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/common/AbstractCallAttempt.java

```diff
@@ -241,6 +243,15 @@ public boolean isRedirect() {
         return redirectAddress != null;
     }
 
+    /**
+     * Returns whether this call attempt is a selector for any of the given functions.
+     * @param functions selectors to match against
+     * @return boolean result
+     */
+    public boolean isSelector(@NonNull final Function... functions) {
+        return Stream.of(functions).map(Function::selector).anyMatch(s -> Arrays.equals(s, this.selector));
```

Comment:
I'm not sure this is repeated enough to affect performance, but just to play it safe, I'd probably use a `for` loop here.

This function definitely improves readability though, good idea. 👍

Reply:
Thanks for the suggestion! Replaced it with for loop.

Reply:
Sad that Java Streams have such high overhead.